package.json

```diff
@@ -38,6 +35,16 @@
     "webpack-node-externals": "^1.7.2"
   },
   "engines": {
-    "node": "12.x"
+    "node": "12.x",
+    "npm": "7.x || 8.x"
+  },
+  "husky": {
+    "hooks": {
+      "commit-msg": "node_modules/.bin/secret-squirrel-commitmsg",
+      "pre-commit": "node_modules/.bin/secret-squirrel",
+      "pre-push": "make verify -j3",
+      "prepare": "npx snyk protect || npx snyk protect -d || true",
+      "preinstall": "[ \"$INIT_CWD\" != \"$PWD\" ] || npm_config_yes=true npx check-engine"
```

Comment:
**Question**:  Why do you use this presinstall task as a hooks. In others migrations this is a task in the script section. Same with prepare.